helm_deploy/manage-hmpps-auth-accounts/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
you can't do both the orb change and kubernetes changes in the same PR.  helm changes need to go to prod before the kubernetes changes can be applied